modules/programs/neovim.nix

```diff
@@ -65,6 +73,16 @@ let
 
   extraMakeWrapperArgs = lib.optionalString (cfg.extraPackages != [ ])
     ''--suffix PATH : "${lib.makeBinPath cfg.extraPackages}"'';
+  extraMakeWrapperLuaCArgs = lib.optionalString (cfg.extraLuaPackages != [ ]) ''
+    --suffix LUA_CPATH ";" "${
```

Comment:
Do the `LUA_CPATH` and `LUA_PATH` variables have to be separated by `;` or can you get away with `:` like the normal `PATH` variable?

Reply:
Yeah, it seems that LUA_PATH expects `;`.  The builtin `getLuaPath` function returns a string separated with `;` when multiple paths are available.